📖 improve ITS, WDS, WEC and registration pages

rishi-jat opened the pull request:
Fixes #2973

Rupam-It commented:
Thanks @rishi-jat for your PR.
I think previously there is PR that is already merged! I'm requesting you to see that. Else Within some time I will review your PR.

rishi-jat commented:
Thanks @Rupam-It for pointing that out!

I'll check the previously merged PR to avoid duplication or conflicts.  
If there's anything redundant or already covered, I'll update this PR accordingly.  
Looking forward to your review as well!

MikeSpreitzer commented:
@rishi-jat: Can you get together with the author of #2997 and make a consolidated proposal? With a preview.

MikeSpreitzer commented on docs/content/direct/its.md:
````diff
@@ -1,10 +1,22 @@
+## Creating an ITS
+1. **Create a ControlPlane for ITS**: Use KubeFlex to create a new ControlPlane of type `vcluster` and run the OCM post-create hook to install OCM components.
+   ```shell
+   kflex create its1 --type vcluster -p ocm
````
You mean use the PostCreateHook named "ocm" defined in the KubeFlex repo? There is more to a KubeStellar ITS. Look at what the core Helm chart does.

rishi-jat commented:
@MikeSpreitzer – I’ve addressed the technical issues you identified in PR #2993:

Fixed:
✅ ITS creation process now correctly shows the core Helm chart as the only supported method
✅ Removed incorrect “OCM machinery” references and clarified KubeStellar’s OCM status add-on
✅ Corrected statement: controllers now clearly described as running in the hosting cluster, not in spaces
✅ Removed WDS content to avoid duplication with PR #3002
✅ Enhanced WEC documentation with clearer lifecycle and management details

The PR now focuses solely on ITS and WEC improvements, with WDS handled in @Tob-iee’s PR #3002 as per your suggestion. Ready for your review. Let me know if anything else needs adjustment!

rishi-jat commented:
Hi @MikeSpreitzer – I’ve successfully resolved the merge conflicts and DCO sign-off issues in PR #2993.

📌 Current PR Focus:

Scope: Documentation improvements for ITS and WEC only

✅ ITS Improvements
	•	Clarified core Helm chart as the only supported creation method
	•	Removed inaccurate references to OCM machinery
	•	Corrected controller placement (runs in hosting cluster)

✅ WEC Enhancements
	•	Expanded documentation on lifecycle and management practices

✅ WDS Coordination
	•	Referenced PR #3002 by @Tob-iee for WDS-related updates

✅ WEC Registration Coordination
	•	Linked to PR #2997 by @Rupam-It for registration flow updates

🔗 PR Coordination Summary:
	•	WDS content → Handled in PR #3002 (@Tob-iee)
	•	WEC registration → Handled in PR #2997 (@Rupam-It)
	•	ITS & WEC docs → Handled in this PR (#2993)

Ready for your review — please let me know if anything further needs adjustment!

rishi-jat commented:
As suggested, I’ll close this PR and open separate ones for ITS and WEC content, and collaborate with others on WEC registration and WDS.
